### ISSUE TITLE
Split up Mensa menu

### ISSUE THREAD

**Issue by Kruemelkatze** [Felfio/AAUFood#19]
Since the new (new) structure, mensa menus are aggregated into one element:

{"starters":[],"mains":[{"name":["Marchfelder Gemüsecremesuppe (A, F, G, L, O) Auflauf mit Kartoffeln, Spinat und Schafkäse (A, C, F, G, L) dazu Paprika Sauerrahmdip (G) und Blattsalat mit Italienischem Dressing (M, O)",_""_],"price":4.7,"isInfo":false},...]}

Furthermore an empty element is added (_""_ above). Starter and Main should be splitted up, the empty tag should be removed.

**Comment by behackl** [comment]
something that could maybe work: separate the content at the first `<br>`, that should split the menu into starter + main dish. for safety, you could also check whether "suppe" or "minestrone" is contained within the first part... :wink:

**Comment by mrukas** [comment]
Split the content at the first "," would not solve the problem i think. For example in "Klare Gemüsesuppe mit Fadennudeln (A, C, F, L)" the first comma is after "(A", so this wouldnt split up the menu at the right position. If you would split up the content on every `<br>` tag, the soup would be in the first place. Maybe we could try it this way.

**Comment by behackl** [comment]
splitting at **every** `<br>` might also not have the desired effect, this would produce

```
Klare Gemüsesuppe mit Frittaten (A, C, F, G, L)
Veganes Chili con Soja mit einem (A, F, L)
Kornspitz (A, F, G, P)
dazu Blattsalat
mit Italienischem Dressing (M, O)
```

yet another thing to try would be to split after every "Allergenkennzeichnung", i.e. after the parentheses. this would split the menu in the actual meals -- is that semantically better than just the separation in starter and main dish?

**Comment by Kruemelkatze** [comment]
FIY: I refactored the Mensa scraping and pushed it to develop. Splitting up the menu is **not** included yet, but the new structure should be easier to comprehend and to edit.

I tried the last suggestion by @behackl of splitting up at **every** `<br>`. This does not look really nice to be honest :)
IMHO the most promising way (first suggestion by @behackl) is to split at the **first** `<br>`, but only if there is "suppe", "minestrone", etc. (--> config parameters) before that. Sounds desperate, but I can not think of any other consistent way.
